print("Submitted!!")
raise Exception("Error works as well!")
